README.md

```diff
@@ -34,9 +34,10 @@ Then install lld from an [LLVM nightly package](http://apt.llvm.org/)
 #### Windows
 Install [MSYS2](http://www.msys2.org/)
 
-In MSYS2 Shell:
+In MinGW64 Shell:
```

Comment:
We also give this guidance on xqemu.com...and I made it very bold because I spent a couple minutes scratching my head as to why I couldn't build anything. So, I think we need to be more clear here too because people (like me) might miss it. Here's what I wrote on xqemu.com:

> After installing MSYS2, you'll need to open MSYS2 MinGW 64-bit to perform building. Otherwise, you may see build failures for cc.exe.

So maybe something along these lines. I'll let you put it in your own words.